phpunit/functional/Glpi/Assets/ImportMapGeneratorTest.php

```diff
@@ -61,22 +61,66 @@ private function setupBasicVirtualFilesystem(): \org\bovigo\vfs\vfsStreamDirecto
                 'modules' => [
                     'Forms' => [
                         'GlpiFormConditionEditorController.js' => '// Some JS content 1',
-                        'GlpiFormDestinationAutoConfigController.js' => '// Some JS content 2'
+                        'GlpiFormDestinationAutoConfigController.js' => '// Some JS content 2',
                     ],
                     'Utils' => [
-                        'HelperFunctions.js' => '// Some helper functions'
-                    ]
-                ]
+                        'HelperFunctions.js' => '// Some helper functions',
+                    ],
+                ],
+            ],
+            'public' => [
+                'lib' => [
+                    'vendor1' => [
+                        'library1.js' => '// Library 1 content',
+                    ],
+                    'vendor2' => [
+                        'library2.js' => '// Library 2 content',
+                    ],
+                ],
+                'build' => [
+                    'compiled.js' => '// Compiled JS content',
+                    'assets' => [
+                        'module1.js' => '// Built module 1',
+                        'module2.js' => '// Built module 2',
+                    ],
+                ],
             ],
             'plugins' => [
                 'myplugin' => [
                     'js' => [
                         'modules' => [
-                            'CustomModule.js' => '// Plugin JS content'
-                        ]
-                    ]
-                ]
-            ]
+                            'CustomModule.js' => '// Plugin JS content',
+                        ],
+                    ],
```

Comment:
Only the `public` directory of the plugins is exposed. It means that the `js` directory here must be placed in the `public` dir of the module.